Wireframe/css/style.css

```diff
@@ -0,0 +1,163 @@
+/* Reset default margin for body element */
+body {
+  margin: 0;
+  background-color: #b4b4b4;
+}
+
+/* Default font styles for body and button elements */
+body, button {
+  font-family: "Source Sans Pro", "Helvetica Neue", Arial, sans-serif;
+  color: #292b2c;
+  font-size: 1rem;
+}
+
+/* Reset margins for heading elements */
+h1, h2, h3, h4, h5, h6 {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+/* Reset margins for paragraphs */
+p {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+/* Header section styles */
+.site-header {
+  display: block;
+  text-align: center;
+  padding: 20px;
+  text-shadow: 2px 2px 4px rgba(255, 255, 255, 0.5);
+  color: #70184b;
+}
+
+/* Main content styles */
+main {
+  max-width: 800px;
+  margin: 0 auto;
+  padding: 2rem;
+}
+
+.main-container {
+  background-color: #f2f2f2;
+  padding: 20px;
+  border: 1px solid #0a0a0a;
+  height: auto;
+  border-radius: 5px;
+  position: relative;
+  font-size: 17px;
+  line-height: 1.5;
+  text-align: justify;
+  margin-bottom: 10px;
+}
+
+/* Main image styling */
+.img-pic {
+  background-size: 100% 100%; 
+  border-bottom: 1px solid #0a0a0a;
+  border-radius: 10px 5px 0 0;
+  width: 798px;
+  height: 400px;
+  position: relative;
+  line-height: 1.5;
+  text-align: justify;
+  margin: -20px;
+  margin-bottom: 10px;
+}
+
+/* Box styling */
+.box {
+  padding: 0;
+  width: 390px;
+  height: 230px;
+  border-bottom: 1px solid #0a0a0a;
+  position: relative;
+  background-size: 100% 100%; 
+  margin: -15px;
+  margin-bottom: 10px;
+}
+
+/* Styles for the inner container */
+.inner, .inner1 {
+  margin-top: 0%;
+  font-size: 15px;
+  line-height: 1.5;
+  padding: 10px;
+  box-sizing: border-box;
+}
+
+/* Article container styles */
+.article-container {
+  display: flex;
```

Comment:
How could you improve these flex declarations to handle the case of very large text or very narrow screens?

Reply:
by adding one of these  
flex-wrap if the content becomes too wide, it might overflow. so I can use flex-wrap to allow flex items to wrap onto multiple lines. or set Min-Width to prevent very narrow screens from squishing the page content or set the overflow: scroll; in case of very large text this will provide a scroll option or make a media queries like this 
@media (max-width: 600px) {
    .article-container {
        flex-direction: column; /* Stack items vertically on narrow screens */
    }
}